components/src/ui-style-constants/typography.ts

```diff
@@ -128,6 +128,6 @@ export const darkLinkPSemiBold = css`
   color: ${COLORS.darkBlack};
 
   &:hover {
-    opacity: 1;
+    opacity: 0.7;
```

Comment:
This should be `1`

Reply:
Probably you can extend `linkPSemiBold` for this.

```suggestion
export const darkLinkPSemiBold = css`
  ${linkPSemiBold}
  color: ${COLORS.darkBlack};
  opacity: 0.7;

  &:hover {
    opacity: 1;
  }
`   
```

Reply:
However, you will need to change the font size since the design font size is `label`. Or you can change the name and font size to `label`.